compiler/rustc_expand/src/mbe/transcribe.rs

```diff
@@ -53,6 +54,20 @@ impl<'a> Iterator for Frame<'a> {
     }
 }
 
+#[derive(SessionDiagnostic)]
+#[error(expand::expr_repeat_no_syntax_vars)]
+struct NoSyntaxVarsExprRepeat {
+    #[primary_span]
+    span: Span,
+}
+
+#[derive(SessionDiagnostic)]
+#[error(expand::must_repeat_once)]
+struct MustRepeatOnce {
+    #[primary_span]
+    span: Span,
+}
```

Comment:
Could you move the diagnostic structs to `rustc_expand::errors`? That's what we've been doing for every other crate so that the errors are off to the side.

Reply:
Solved in the upcoming commit